.drone.yml

```diff
@@ -4,20 +4,37 @@ type: docker
 name: default
 
 steps:
-- name: install, build doc run scripts
-  image: python:3.7
+- name: check code complexity
+  image: python:3.9
   commands:
-  - git fetch --tags 
-  - python setup.py install
+  - pip install radon pylint
+  - ./code_quality.sh
+  - python code_pylint.py
+
+- name: check pep8 formatting
+  image: python:3.9
+  commands:
+    - git fetch --tags
+    - pip3 install -U pip autopep8
+    - bash code_pep8.sh
 
-- name: develop
-  image: python:3.7
+- name: install, build doc run scripts
+  image: python:3.8
   commands:
-  - python setup.py develop
+  - git fetch --tags
+  - pip install Cython sphinx sphinx_rtd_theme coverage
+  - python setup.py install
+  - cd scripts
```

Comment:
There is no scripts in git. Forgot to add some files to git?

Reply:
Actually, I added ci_scripts and some scripts files in the next PR #2